### PR TITLE
WRP-3071: Updated dependencies to fix vulnerabilities

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to fix vulnerabilities

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated dependencies to fix vulnerabilities

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3071

### Comments
Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: packages/theme/template/package.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
     "react": "^18.2.0"
   },
   "peerDependencies": {
-    "ilib": "^14.14.0 || ^14.14.0-webos1"
+    "ilib": "^14.15.1 || ^14.15.1-webos1"
```
Comment (daniel-stoian-lgp) [inline packages/theme/template/package.json:41]: latest version of ilib-webos is 14.15.1 https://www.npmjs.com/package/ilib-webos?activeTab=versions

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Comment by hong6316** [comment]
Before updating the version of ilib, you must also update the version of [ilib-webos](https://www.npmjs.com/package/ilib-webos?activeTab=versions).
If ilib-webos does not have the same version, please do not update and keep the version of ilib.

**Review by hong6316 — APPROVED** [review]
👍

File: packages/theme/template/package.json (hunk 1 above)
Comment (hong6316) [reply packages/theme/template/package.json:41]: Good, we need to check this.

**Pull request merged**